packages/core/src/browser/common-frontend-contribution.ts

```diff
@@ -989,9 +989,15 @@ export class CommonFrontendContribution implements FrontendApplicationContributi
             // Input control colors should be aligned with https://code.visualstudio.com/api/references/theme-color#input-control
             // if not yet contributed by Monaco, check runtime css variables to learn
             { id: 'input.border', defaults: { hc: 'contrastBorder' }, description: 'Input box border.' },
-            { id: 'inputValidation.errorForeground', defaults: { dark: 'errorForeground', light: 'errorForeground', hc: 'errorForeground' }, description: 'Input validation foreground color for error severity.' },
-            { id: 'inputValidation.infoForeground', description: 'Input validation foreground color for information severity.' },
-            { id: 'inputValidation.warningForeground', description: 'Input validation foreground color for warning severity.' },
+            { id: 'inputValidation.infoBackground', defaults: { dark: '#063B49', light: '#D6ECF2', hc: Color.black }, description: 'Input validation background color for information severity.' },
```

Comment:
Where did you get such colors? Is it aligned with VS Code codebase?

Reply:
@akosyakov the colors come directly from vscode:
https://github.com/microsoft/vscode/blob/88f63bcd875d986c616b4bc65bc7a590469985c7/src/vs/platform/theme/common/colorRegistry.ts#L208-L216

Reply:
These colors are already registered by Monaco we don't need to duplicate them.

Reply:
Although, it looks bogus that we redeclare foreground. I think it was overlooked as well and should be removed.

Reply:
I think we need a runtime check that we avoid redeclaring Monaco variables.

Reply:
> These colors are already registered by Monaco we don't need to duplicate them.

I understand, color should only be declared if they are not already contributed by Monaco.
I tried to find some standard documentation on coloring provided by monaco and could only find the following [doc](https://github.com/microsoft/monaco-editor/blob/daabff0021fd6d4f5b57479b99b9a8ab900f951a/test/playground.generated/customizing-the-appearence-exposed-colors.html#L82-L89). Do you know of a better place to look or is it sufficient?

Reply:
I look at VS Code codebase and then check in the dev tools whether we have a corresponding file and color. Unfortunately there is no other way.

I think we could add a check in `MonacoColorRegistry` which log with an error if someone tries to  override predefined Monaco colors and does nothing more.